src/components/content/deployedServices/myServices/MyServices.tsx

```diff
@@ -178,18 +178,34 @@ function MyServices(): React.JSX.Element {
         return [
             {
                 key: 'details',
-                label: (
-                    <Button
-                        onClick={() => {
-                            handleMyServiceDetailsOpenModal(record);
-                        }}
-                        className={'button-as-link'}
-                        icon={<InfoCircleOutlined />}
-                        type={'link'}
-                    >
-                        details
-                    </Button>
-                ),
+                label:
+                    record.serviceDeploymentState === DeployedService.serviceDeploymentState.DESTROY_SUCCESSFUL ||
+                    record.serviceDeploymentState === DeployedService.serviceDeploymentState.DEPLOYMENT_FAILED ? (
```

Comment:
When service state is `deploy failed`, the operation `details` should be enable for user to view error message. I fix this error with PR https://github.com/eclipse-xpanse/xpanse-ui/pull/456